### PR TITLE
Fix V2 Issues

### PR DESCRIPTION
**Pull request by x4d3**

### CODE REVIEW

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invites_controller.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ def send_org_invite(invite)
       # Generate token if not generated
       user.send(:generate_confirmation_token!) if !user.confirmed? && user.confirmation_token.blank?
 
-      MnoEnterprise::SystemNotificationMailer.organization_invite(invite).deliver_later
+      MnoEnterprise::SystemNotificationMailer.organization_invite(invite).deliver_now
```
Comment (ouranos) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invites_controller.rb]: Why remove the `deliver_later`? By default it'll be performed now but if we add sidekiq it'll automatically use it.

File: api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb [hunk 2]
```diff
@@ -100,12 +100,13 @@ def user_params
...
+      user.password = Devise.friendly_token
+      user.save!
 
-      # Reset the confirmation field so we can track when the invite is send - #confirmation_sent_at is when the confirmation_token was generated (not sent)
-      # Not ideal as we do 2 saves, and the previous save trigger a call to the backend to validate the token uniqueness
-      user.attributes = {confirmation_sent_at: nil, confirmation_token: nil}
-      user.save
+      # # Reset the confirmation field so we can track when the invite is send - #confirmation_sent_at is when the confirmation_token was generated (not sent)
+      # # Not ideal as we do 2 saves, and the previous save trigger a call to the backend to validate the token uniqueness
+      # user.attributes = {confirmation_sent_at: nil, confirmation_token: nil}
```
Comment (ouranos) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb]: Why is this commented? Do we still have the same behavior?

Comment (x4d3) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb]: Well spotted, I was doing some test, and I did not check if it was the same behavior.

File: api/app/controllers/mno_enterprise/jpi/v1/admin/invites_controller.rb (hunk 1 above)
Comment (x4d3) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/invites_controller.rb]: Invite is not serializable. Should we replace it to take invite_id instead ?

**Pull request merged**